include/crm/common/options_internal.h

```diff
@@ -79,7 +79,6 @@ enum pcmk__opt_flags {
 
 typedef struct pcmk__cluster_option_s {
     const char *name;
-    const char *alt_name;
```

Comment:
Skip this commit, we'll likely rename options in the future (e.g. s/orphan/removed/, s/stonith/fencing/, s/pe/scheduler/)

Wouldn't be a bad idea to do those for 3.0.0 ...

Reply:
Couldn't we just do schema transforms for those instead of maintaining support for the old options? Seems much cleaner...

Reply:
We need to accept the old names (even in new CIBs) for a long while since they will be in documentation, how-to's, scripts, high-level tools, etc.

Reply:
:'(